src/screens/FeedScreen/RenderList.js

```diff
@@ -217,7 +232,8 @@ const RenderListFeed = (props) => {
             statusVote={voteStatus}
             showScoreButton={showScoreButton}
             onPressScore={() => showScoreAlertDialog(item)}
-            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
+            isSelf={isSelf}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Footer` component now receives new props: `isSelf` and `isShowDM`. Ensure that these props are handled correctly in the `Footer` component.
```diff
+ isSelf={isSelf}
+ isShowDM
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->